Source/WebKit/UIProcess/ViewSnapshotStore.cpp

```diff
@@ -110,6 +111,16 @@ void ViewSnapshotStore::discardSnapshotImages()
         m_snapshotsWithImages.first()->clearImage();
 }
 
+void ViewSnapshotStore::discardSnapshotImagesForOrigin(const WebCore::SecurityOriginData& origin)
+{
+    for (auto it = m_snapshotsWithImages.begin(); it != m_snapshotsWithImages.end();) {
+        auto currentIterator = it;
```

Comment:
Maybe clearer to say "auto viewSnapshot = *it" here.

Reply:
Ok.